### ISSUE TITLE
json label creation fails for validation sets

### ISSUE THREAD

**Issue by harryjmoss** [UCL-RITS/vetii#5]
`convert_masatixml_to_json_labels.py` creates validation annotation files that also contain the training set annotations
